server/code/setup/cpdb

```diff
@@ -131,6 +131,18 @@ class PostgresqlSetup(DbSetup):
             # Append / for the database name:
             self.jdbc_url = "%s/" % (self.jdbc_url)
         self.jdbc_url = "%s%s" % (self.jdbc_url, db)
+
+        # Appending SSL connection details
+        if ssl == True and sslfactory is None and sslcertpath is None:
```

Comment:
This block can be rearranged such that you're only checking the ssl variable once:
```
if ssl == True:
  if sslfactory is None and sslcertpath is None:
    ...
  elif ...
    ...
  elif ...
```

Reply:
Updated as suggested.